_layouts/package.html

```diff
@@ -20,6 +20,8 @@
   </div>
 </div>
 
+<script src={{ "/js/contribution_suggestions.js" | prepend: site.baseurl }}></script>
```

Comment:
I don't believe you accomplish anything by including this script load at the top of the page, rather than at the bottom with package_body_tabs.js. I believe it would be cleaner to keep all of the script loads together at the bottom of the page, after the DOM definitions.